envoy/tests/docker/default/Dockerfile-service

```diff
@@ -0,0 +1,10 @@
+FROM envoyproxy/envoy-alpine:latest
```

Comment:
why this is not parametrised with the envoy version?

Reply:
To my knowledge this actually doesn't matter because we don't get metrics from here.